libs/core/models/animeDetail.ts

```diff
@@ -0,0 +1,43 @@
+import { OmitImmerable } from '../models/immerable';
+
+import { Anime } from './anime';
+import { Genre } from './genre';
+import { Studio } from './studio';
+
+/** Base model for anime. */
+export class AnimeDetail extends Anime {
+
+  /** Youtube trailer. */
+  public readonly trailerYoutubeId: string | null;
+
+  /** Airing at the moment. */
+  public readonly airing: boolean;
+
+  /** Synopsis. */
+  public readonly synopsis: string;
+
+  /** List studio id. */
+  public readonly studios: readonly number[];
```

Comment:
I suggest using **indexed access type**, that way if studio id would change from number to string, you wouldn't need to change it everywhere by hand.
```suggestion
  public readonly studios: readonly Studio['id'][];
```